### ISSUE TITLE
Move PythonTemplate to GitHub

### ISSUE THREAD

**Issue by tdowrick** [UCL/scikit-surgery#58]
I think that we decided that we want to move PythonTemplate to GitHub (correct?)

Are we going to move to GitHub Actions as the default CI?
Should we keep the GitLab CI template in, for any private projects that we want to keep on GitLab? 
We should add github.com/UCL as an option for `repository_server/repository_path`
Anything else?

Can discuss in more detail at the next SNAPPY meeting.

@mianasbat @MattClarkson @thompson318

**Comment by thompson318** [comment]
Yes, I think GitHub Actions should be the default and leave GitLab CI template in because I think people will still want to keep private repositories on gitlab. 

Now we've got the paper out I think we should add a note about citing it too, In scikit-surgeryfred I put a citing section in the readme. We could put this in the template.

**Comment by tdowrick** [comment]
From dev meeting:

Keep gitlab CI for people who want private repos on WeissLab.
Set default repo location to GitHub
Don't make it sound like it is only for WEISS projects - check over text for things that can be removed

**Comment by tdowrick** [comment]
- [x] Files/git history moved
- [ ] Issues moved 
- [ ] Register project with coveralls, add coveralls.yml file, add coveralls and pyyaml to requirments-dev
- [ ] CI working for Windows/Linux, using GitHub Actions. (Skipping Mac CI, as there are issues with GPU)
- [ ] Old CI files removed
- [ ] Badges updated
- [ ] Add PYPI_API_TOKEN as Secret and Test deploy to PyPI 
- [ ] Update read the docs
- [ ] Relevant links changed to point to GitHub rather than GitLab
- [ ] Changes merged
